providers/src/airflow/providers/google/cloud/hooks/dataproc.py

```diff
@@ -118,11 +118,11 @@ def add_args(self, args: list[str] | None = None) -> None:
 
     def add_query(self, query: str) -> None:
```

Comment:
Don't you want maybe to allow a list of queries (`query: str | list[str]`)? it will be nicer to handle rather than repeating:
```python
hook.add_query("query1")
hook.add_query("query2")
.
.
.
```

Also, could you please merge/rebase `main`?

Reply:
agree, updated